CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Contributing
+
+For general contributing guidelines, see the [Contributing Guide on DoneJS.com][].
+
+## Developing Locally
+
+### Setup
+
+Fork and clone this repository. See [Fork A Repo][] and [Cloning a repository][] for help.
+
+Install the packages defined in [`package.json`][]:
+
+```
+npm install
+```
+
+### Build
+
+There is no build step for this repo.
+
+### Test
+
+Tests are located in `test.js`, at the root of this repo.
+
+To run the tests, execute the `test` script defined in [`package.json`][]:
+
+```
```

Comment:
The point of not including "shell" is that it's NOT the correct highlighting. If you really want the pretty (inconsistent) colors then okay, but realize it is not correct because this is not shell code.